modules/table/src/main/java/org/apache/ignite/internal/table/distributed/StorageUpdateHandler.java

```diff
@@ -81,8 +89,31 @@ public void handleUpdate(
             TablePartitionId commitPartitionId,
             @Nullable ByteBuffer rowBuffer,
             @Nullable Consumer<RowId> onReplication
+    ) {
+        handleUpdate(txId, rowUuid, commitPartitionId, rowBuffer, onReplication, null);
+    }
+
+    /**
+     * Handles single update.
+     *
+     * @param txId Transaction id.
+     * @param rowUuid Row UUID.
+     * @param commitPartitionId Commit partition id.
+     * @param rowBuffer Row buffer.
+     * @param onReplication Callback on replication.
+     * @param lowWatermark GC low watermark.
+     */
+    public void handleUpdate(
+            UUID txId,
+            UUID rowUuid,
+            TablePartitionId commitPartitionId,
+            @Nullable ByteBuffer rowBuffer,
+            @Nullable Consumer<RowId> onReplication,
+            @Nullable HybridTimestamp lowWatermark
```

Comment:
You need to create a ticket (and add TODO) to support this argument with write commands or something like that.